pkg/eventual/value.go

```diff
@@ -0,0 +1,141 @@
+package eventual
```

Comment:
Should we call it `eventually` instead? Seems like adverbs or usually preferred over adjectives for package names.